purty/README.md

```diff
@@ -0,0 +1,24 @@
+# Purty
+
+Currently Purty is a little difficult to build with nix for two reasons:
```

Comment:
I don't suppose we can get it from nixpkgs, can we? I'm seriously considering adding a `nixpkgs-unstable` pin for a few tools that we use for the docs, is purty in `nixpkgs-unstable`?

Reply:
no, it will likely be quite a while before a stable purty is in nixpkgs, it's currently quite a young and fast-moving project.